src/App.js

```diff
@@ -77,19 +77,20 @@ export default class App {
 
     // retrieve all (unique) queries
     const qs = routes.map(route => {
-      log(`${route.handler.name} queries: %o %o %o %o`, route, route.handler, route.handler.prototype, route.handler.queries ? route.handler.queries : 'no qs');
-      return route.handler.queries;
-    }).filter(v => v).reduce((ac, q) => ({
-      ...ac,
-      ...q
-    }), {});
+        log(`${route.handler.name} queries: %o %o %o %o`, route, route.handler, route.handler.prototype, route.handler.queries ? route.handler.queries : 'no qs');
+        return route.handler.queries;
+      })
+      .filter(v => v)
+      .reduce((ac, qs) => ({  ...ac, ...(
+        Object.keys(qs)
```

Comment:
at this point they all have this uniform shape here. See https://github.com/buildo/revenge/blob/revenge-84-queriesfilter_should_be_perquery/src/decorators/queries.js